src/main/java/org/apache/commons/collections4/CollectionUtils.java

```diff
@@ -1112,6 +1093,7 @@ public int hashCode() {
      * @return a boolean indicating whether the collection has changed or not.
      * @throws NullPointerException if the collection or iterator is null
      */
+    @SuppressWarnings("unchecked")
```

Comment:
Of cource, it is minnor. this is not necessary, but if it is applied, warnings about not correct cast will disappear. I talked about this code: `collection.addAll((Collection<? extends C>) iterable)`